lib/test/t_Study_saveload.cxx

```diff
@@ -59,6 +59,17 @@ int main(int argc, char *argv[])
   try
   {
 
+    // Create a Study Object by giving its name
+    {
+      Study study(fileName);
+    }
+
+    // Create a Study Object by giving its storage manager with a nonzero compression level
+    {
+      Study study;
+      study.setStorageManager(XMLStorageManager(fileName, 5));
+    }
+
```

Comment:
@dbarbier It is just to check that it is possible to create an XMLStorageManager using a given compression level and to set it into a Study instance.